pennylane/operation.py

```diff
@@ -110,6 +110,19 @@
 from .utils import pauli_eigs
 
 
+def __getattr__(name):
+    warning_names = {"Sample", "Variance", "Expectation", "Probability", "State", "MidMeasure"}
+    if name in warning_names:
+        obj = getattr(qml.measurements, name)
+        warning_string = f"qml.operation.{name} is deprecated. Please import from qml.measurements.{name} instead"
```

Comment:
As this is now a deprecation warning, it would be good to adjust the changelog accordingly (I think the import path change might be listed as a breaking change).